tests/test_point_density.py

```diff
@@ -149,6 +160,52 @@ def test_point_density_ppf(scale: Scale):
     assert uniform_dist.ppf(1) == pytest.approx(scale.high, rel=0.1)
 
 
+@pytest.mark.parametrize(
+    "scale", [Scale(-3, 10), LogScale(0.01, 5, 500)],
+)
+def test_point_density_modes(scale: Scale):
+    dist = get_dist_from_scale(scale)
+
+    modes = dist.modes()
+
+    assert len(modes) > 0
+
+    mode_densities = np.array([dist.pdf(mode) for mode in modes])
+
+    assert np.all(mode_densities == mode_densities[0])
+
+    mode_density = mode_densities[0]
+
+    xs, _ = dist.to_arrays()
+
+    all_densities = np.array([dist.pdf(x) for x in xs])
+
+    assert mode_density == np.max(all_densities)
```

Comment:
why?

(I think it's bad to make the test more permissive if it can instead be more restrictive)

Reply:
Because I think you're trying to say "These two floating point numbers should be equal" and [pytest.approx](https://docs.pytest.org/en/stable/reference.html#pytest-approx) with default settings seems like a reasonable way to state that. If you use exact equality you're stating something else (something like "These two floating point numbers should be equal and also the result of the same sequence of operations"). The test might break when someone (us, or numpy authors) changes the implementation and we'll have to look at the test and think about which of the two equalities you intended when you wrote the test.

Reply:
oh cool, yeah, totally agree!

That pytest.approx doc page was illimunating